packages/@okta/vuepress-site/docs/reference/csi-delauth-hook/index.md

```diff
@@ -0,0 +1,541 @@
+
+The following sections describes each of these operations about the objects in their requests and responses.
+
+## User attributes
+
+In oder to support user lifecycle and credential lifecycle, we expect the external source to provide certain attributes:
+
+- `status`: The status of the user, with value ACTIVE or DISABLED.
+- `passwordExpiryTime`: The time when the user's password will expire, in the form of Epoc time.
+- `lastUpdate`: When the user's profile was last updated in the external source
+
+## Delegated authentication request
+
+This type of request has the following requestType:
+
+- `user.authenticate`
+
+### Objects in the request from Okta
+
+The outbound call from Okta to your external service will include the following objects in its JSON payload:
+
+#### data.context.credential
+
+Provides two name-value pairs, one for identifying the subject user to authenticate, and the other for providing the password of that user.
+
+#### data.action
+
+The current default action that Okta will take. Okta will take this action if your external service sends an empty HTTP 204 response. You can override the default action by returning a `commands` object in your response specifying the action to take.
+
+
+### Objects in Response You Send
+
+The `commands` and `error` objects that you can return in the JSON payload of your response are defined as follows:
+
+#### commands
+
+The `commands` object is where you can provide commands to Okta. It is an array, allowing you to send multiple commands. Each array element needs to consist of the following name-value pair:
+
+| Property | Description                                           | Data Type       |
+|----------|-------------------------------------------------------|-----------------|
+| type     | One of the [supported commands](#supported-commands). | String          |
+| value    | The parameter to pass to the command.                 | [value](#value) |
+
+##### Supported Commands
+
+The following command is supported for the delegated authentication request:
+
+| Command                         | Description                                                                                                              |
+|---------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| com.okta.action.update          | Specify the result of the authentication, either success or one of the causes of failure. |
+
+
+##### value
+
+The `value` object is the parameter to pass to the command. For the `com.okta.action.update` command, `value` should be an object containing a `credential` property set to one of the following values:
+
+- `VERIFIED`: The user is authenticated successfully
+- `UNVERIFIED`: The user fails to authenticate due to invalid credential
+- `ACCOUNT_LOCKED`: The user account is locked
+- `ACCOUNTOUN_DISABLED`: The user account is disabled
```

Comment:
Is this value correct? `ACCOUNT_DISABLED` instead?

Reply:
good catch - should be `ACCOUNT_DISABLED`.

Reply:
Would you please generate a change request so that I can use to replace since other changes are handled in that way?

Reply:
Sorry, missed this, and I see you fixed it. Resolving.